### PR TITLE
OV-4: Hide Default Password input Reveal Functionality

### PR DESCRIPTION
**Pull request by lfelix3011**
This Pull Request is to solve this review https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/issues/4#issuecomment-2308097788

Test did in edge browser
image

### CODE REVIEW

File: frontend/src/bundles/common/components/input/input.tsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ type Properties<T extends FormValues> = {
     name: FieldInputProps<T>['name'];
     placeholder?: string;
     icon?: 'right' | 'none';
+    sx?: object;
```
Comment (JKaypa) [inline frontend/src/bundles/common/components/input/input.tsx:18]: A better type for sx is : SystemStyleObject

Comment (lfelix3011) [reply frontend/src/bundles/common/components/input/input.tsx:18]: Thanks

**Review by o-nedashkivska — DISMISSED** [review]

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/auth/components/common/password-input/password-input.tsx
[the remark this answers is not in the document]
Comment (lfelix3011) [reply frontend/src/bundles/auth/components/common/password-input/password-input.tsx:1]: @nikita-remeslov already remove the styles from Password component and put it in global style, i am not sure if soudl be in global o create a style of reset 
image

File: frontend/src/framework/theme/styles/global.styles.ts [hunk 2]
```diff
@@ -12,6 +12,9 @@ const global = {
         flexGrow: 1,
         flexDirection: 'column',
     },
+    'input::-ms-clear, input::-ms-reveal': {
```
Comment (nikita-remeslov) [inline frontend/src/framework/theme/styles/global.styles.ts:15]: Add a comment what these styles do

**Review by nikita-remeslov — APPROVED** [review]

**Review by sofiia-trokhymchuk — APPROVED** [review]

**Pull request merged**